platform/features/table/src/controllers/MCTTableController.js

```diff
@@ -284,6 +284,20 @@ define(
         };
 
         /**
+         * Given a value, if it can be coerced to a number, then return a
+         * number representation. It's a little more robust than using just
+         * Number() or parseFloat, or isNaN in isolation, all of which are
+         * fairly inconsistent in their results.
+         * @param value The value to cast (if possible)
+         * @returns {*} The value cast to a Number, or the original value if
+         * a Number representation is not possible.
+         */
+        MCTTableController.prototype.toNumber = function (value){
```

Comment:
It looks like this is exposed externally such that it can be tested, but wouldn't it be better to test the sortComparator itself?  Seeing as this function is only utilized immediately before passing the results to `sortComparator`, it would make sense to inline this function inside of the sortComparator.  Then your test cases can focus on the expected sorting behavior instead of the implementation.